_opcodes/op245.html

```diff
@@ -10,4 +10,4 @@
 iwd2: 0
 pst: 0
 ---
-This effect enables a 1% chance of the targetted creature(s) going berserk on each attack it makes. This affect sets the berserk stat changes and then attaches <a href="#op246">effect 246</a>.
+Aside from setting <a href="../files/ids/bgee/stats.htm#142">CHECKFORBERSERK</a>, this effect (or that STAT) do nothing (needs more research...?).
```

Comment:
Where is this from? From how we use it, I'd say the opcode and stat are aptly named.

Reply:
> Where is this from? From how we use it, I'd say the opcode and stat are aptly named.

OK, restored (also op246), but flagged as non-EE behavior.

Reply:
I doubt they changed it. When reverse engineering by testing It's just hard to see where the border between the opcode and core code is. So I agree with the description that the opcode only sets the stat, but that does trigger other things in core.